stonesoup/types/tests/test_orbital.py

```diff
@@ -54,28 +54,41 @@
 import pytest
 from datetime import datetime
 
-from ...types.array import StateVector
+from ...types.array import StateVector, StateVectors
 from ..orbitalstate import OrbitalState
 
 # Time
 dtime = datetime.now()
 
 # Orbital state vector in km and km/s
 orb_st_vec = StateVector([-6045, -3490, 2500, -3.457, 6.618, 2.533])
+# Initialise an equivalent StateVectors object
+orb_st_vec2 = StateVector([-3756.52, 5626.22, 488.986, -4.20561, -2.29107, -5.98629])
+orb_st_vecs = StateVectors([orb_st_vec, orb_st_vec2])
+
 cartesian_s = OrbitalState(orb_st_vec, coordinates='Cartesian')
 # ensure that the Gravitational parameter is in km^3 s^-2
```

Comment:
The orbital state docs state that the gravitational parameter should be in m^3 s^-2 and the default is also in m^3 s^-2, should this also be m^3 insteaad of km^3?

Reply:
No, the input (line 66) is in [km] so GM has to be in units of [km^3 s^-2]. Since its default is in [m^3 s^-2] it gets multiplied by 1e-9 (km/m)^3. Subsequent calculations work themselves out.